util/regexp-assemble/data/922110-chain1.data

```diff
@@ -0,0 +1 @@
+920600.data
```

Comment:
Symlink support on Windows is brittle and requires "developer mode". See https://stackoverflow.com/questions/5917249/git-symbolic-links-in-windows/59761201#59761201.
I'd rather duplicate 920600.data, add a comment to the new file and wait for the include processor to be able to handle that.

Reply:
Fine by me. Duplicated the file and add a comment.